src/profile/EditProfile.tsx

```diff
@@ -56,7 +67,7 @@ const EditProfile = (props: any) => {
     // what we did above, was the get the user id from the navigation bar
     const response = await RestAPI.get("pareto", `/users/${userId}`, {});
     // here we are populating our initial state. In the future, we will likely just pass stuff in via props, instead of running a fresh network request. That was a legacy decision, don't worry about it @antonio-b
-    console.log('u', response);
+    console.log("u", response);
```

Comment:
Delete this console.log, easy to forget, believe me I know 😄

Reply:
Done